database_setup.rb

```diff
@@ -0,0 +1,63 @@
+require 'pg'
+
+class DatabaseSetup
```

Comment:
create a separate class for database connection, use class variable https://syntaxdb.com/ref/ruby/class-variables